components/FooterSSR/index.jsx

```diff
@@ -0,0 +1,16 @@
+import { Container } from '@catho/quantum';
```

Comment:
The "FooterSSR" folder seems a little redundant to me, considering that we are inside the ssr-simulator project. What do you think about just putting "Footer"?